### ISSUE TITLE
Setup Watchla - 

### ISSUE THREAD

**Issue by Slidevalve** [itsMeDavidV/Watchla-Support#58]
Installed Watchla per instructions.  However, it has not shown up on my Watch “Install” list.  Refreshed watch.  Restarted iPhone.  Uninstalled and reinstalled on iPhone.  Reset all devices and did all of the above again.  Still no Watchla listed on watch app.  Next step?

**Comment by itsMeDavidV** [comment]
@Slidevalve By what you described, I'd venture to say you might be experiencing pairing issues between your iOS device and Apple Watch. 

Here's a helpful article on how to detect and address pairing issues with your Apple Watch: https://support.apple.com/en-us/HT205025

Normally in this scenario I would recommend uninstalling Watchla, restarting your iOS device and Apple Watch, then re-install Watchla on your iOS device and the watch app should populate on your iOS device's "Watch" app. However it seems you have done that.

Alternatively, you can download Watchla from the App Store app on your Apple Watch. However, if you are experiencing pairing issues as described above then the Watchla watch app and iOS app will not be able to communicate which will prevent you from signing in.

If all else fails, I would recommend re-pairing your Apple Watch to your iOS device as a last resort.

**Comment by itsMeDavidV** [comment]
@Slidevalve given the time since last response, I'm going to assume this was resolved and close the issue. If the issue is still occurring, please feel free to re-open and reach out.

**Comment by Slidevalve** [comment]
Actually, I tried all of the recommended items again, including resetting my watch to factory and starting all over.  I deleted many of the watch apps just to make sure that was not an issue.  I started from scratch and reloaded your app.  I still does not show up on my iPhone in the Apple Watch app as ready to install.  All the other watch apps I took off are in there ready to be installed.  So, I am not sure what to do next. 

Dennis A. Nease, AIA
WNB Architects, Inc.

**Comment by itsMeDavidV** [comment]
@Slidevalve Are you able to install any of the other watch apps listed or does installation time out?

Also have you tried installing the Watchla watch app from the App Store app on your Apple Watch?

**Comment by Slidevalve** [comment]
Just installed weather app to watch from phone successfully. 

Dennis A. Nease, AIA

**Comment by itsMeDavidV** [comment]
@Slidevalve Does installing Watchla from the App Store on your Apple Watch not work?

**Comment by Slidevalve** [comment]
I may be dense…. But my Apple Watch doesn’t have that app.  What am I missing.  

Dennis A. Nease, AIA
WNB Architects, Inc.

**Comment by itsMeDavidV** [comment]
@Slidevalve no worries it can be easy to miss. If you have WatchOS 6.0 or later the App Store should be on your Apple Watch next to all the other ones. I attached a screenshot for reference.

Once you find it, just search for Watchla and tap install.

![image](https://user-images.githubusercontent.com/12491192/164278583-e3a96b64-fcaf-4b46-a344-2091300162a6.png)

![image](https://user-images.githubusercontent.com/12491192/164278641-228d9a3f-6750-4ea2-b887-b49595fcaadb.png)

**Comment by Slidevalve** [comment]
This may the reason no App Store on my watch. 


I’ve followed all on line suggestions to install App Store on watch. None work or match what is on my phone or watch.  

Dennis A. Nease, AIA

**Comment by itsMeDavidV** [comment]
@Slidevalve Have you checked to see which version of WatchOS your Apple Watch is on?

You can check by going to the "Watch" app on your iOS device, then goto "General" -> "About" and then check the "version" value

**Comment by itsMeDavidV** [comment]
Hi @Slidevalve Is this issue still occurring for you?

**Comment by Slidevalve** [comment]
Yes.  No luck.  Last email I sent showed how I was not able to find the APP Store on my Apple Watch and why I thought it might be the issue (early generation watch).  To be honest, I gave up.  I am sort of tech savvy but this one has me stumped. 

Dennis A. Nease, AIA
WNB Architects, Inc.

**Comment by itsMeDavidV** [comment]
@Slidevalve Sorry to hear that. I think first step would be confirming your WatchOS version. It would need to be 6.2.6 or above to download Watchla and 6.2 or above to download the App Store app.

If you meet the version requirements and still can't download the App Store app or the Watchla app, I'd recommend reaching out to Apple Support as they should be happy to help with this since it's an OS issue.

**Comment by Slidevalve** [comment]
Yes.  I attached that screen clip.  It is:

Version 4.3.2, Model NJT2LL/A. 

Dennis A. Nease, AIA
WNB Architects, Inc.
